numero.Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM golang:1.17.6-alpine3.15 AS dependencies
+
+RUN apk update && apk add --no-cache \
+  ca-certificates \
+  musl-dev \
+  gcc \
+  build-base \
+  git
+
+ENV GO111MODULE=on \
+  CGO_ENABLED=1 \
+  GOOS=linux \
+  GOARCH=amd64
+
+WORKDIR /build
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+FROM dependencies AS builder
+
+COPY . .
+
+RUN ./build.sh
+
+WORKDIR /dist
+
+RUN cp /build/main .
+
+FROM alpine:3.15 as dist
+COPY custom_account_setup.cdc ./
```

Comment:
Add three of these -- one each referencing the environment of `custom_account_setup_emulator.cdc`, `custom_account_setup_testnet.cdc`, `custom_account_setup_mainnet.cdc` and then each environment must have the `FLOW_WALLET_SCRIPT_PATH_CREATE_ACCOUNT` set to reference which script should be used in which environment when creating new accounts.

Reply:
Instead of splitting the custom setup scripts out by Flow network, I've split them by environment, so that we can distinguish the Electables account we're using for QA and Staging. 

The files are:
`custom_account_setup_emulator.cdc`
`custom_account_setup_qa.cdc`
`custom_account_setup_staging.cdc`
`custom_account_setup_production.cdc`

I also updated the env vars in heroku.